### PR TITLE
EMP-82 Add new optional to make sure paginationDelta is pulled from configuration

### PR DESCRIPTION
**Pull request by alecsloan**

### CODE REVIEW

**Comment by alecsloan** [comment]
@marco-leo let me know if you'd rather me replace the cpSearchResultsPortletInstanceConfiguration than add an extra check.

**Comment by alecsloan** [comment]
ci:test

File: commerce-product-content-search-web/src/main/java/com/liferay/commerce/product/content/search/web/internal/portlet/CPSearchResultsPortlet.java [hunk 1]
```diff
@@ -171,8 +173,22 @@ protected void paginate(
 		Optional<Integer> paginationDeltaOptional =
 			paginationDeltaParameterValueOptional.map(Integer::valueOf);
 
+		int configurationPaginationDelta =
+			cpSearchResultsPortletInstanceConfiguration.paginationDelta();
+
+		Optional<PortletPreferences> portletPreferencesOptional =
+			portletSharedSearchSettings.getPortletPreferences();
```
Comment (ethanbustad) [inline commerce-product-content-search-web/src/main/java/com/liferay/commerce/product/content/search/web/internal/portlet/CPSearchResultsPortlet.java:180]: @alecsloan @marco-leo note that this line is incompatible with 7.2 -- the compatible method is `portletSharedSearchSettings.getPortletPreferences71`, but we haven't merged that change yet... so not sure if we want to address it here or in the other pull. Just FYI.

**Comment by ethanbustad** [comment]
ci:test
Tests may pass now that https://github.com/liferay/com-liferay-commerce/pull/1436 is merged in.

**Comment by alecsloan** [comment]
ci:test

File: commerce-product-content-search-web/src/main/java/com/liferay/commerce/product/content/search/web/internal/portlet/CPSearchResultsPortlet.java (hunk 1 above)
Comment (marco-leo) [reply commerce-product-content-search-web/src/main/java/com/liferay/commerce/product/content/search/web/internal/portlet/CPSearchResultsPortlet.java:180]: @ethanbustad let's update the other pull, as I do think we will nee to fork for 7.2

**Pull request merged**